parse.go

```diff
@@ -178,6 +222,18 @@ func process(specs []*spec, args []string) error {
 		if spec.short != "" {
 			optionMap[spec.short] = spec
 		}
+		if spec.env != "" {
+			var value string
+			var found bool
+			value, found = os.LookupEnv(spec.env)
+			if found {
```

Comment:
Can you collapse these four lines into `if value, found := os.LookupEnv(spec.env); found {`?